bin/boom

```diff
@@ -145,6 +156,16 @@ if __name__ == '__main__':
         # conf['pipeline']['rabbitmq_host'] = rabbitmq.attrs['NetworkSettings']['IPAddress']
         conf['pipeline']['rabbitmq_host'] = '127.0.0.1' # Using host mode for the network
 
+    elif conf['pipeline']['mode'] == 'aws':
+        tf = Terraform(working_dir='../../boom/infra')
```

Comment:
This should be an arg that users can set when calling BOOM. You can add a new option to the binary.

Reply:
The way I have designed it is to keep it abstracted from the user. The user need not worry how everything is happening and that is the reason why currently I am not taking this as an input from the user.
Also, my bad, this should have not been in this branch. This should come with the aws code

Reply:
User does not need to work in the same directory, so that the relative path would definitely fail.

Reply:
Ok thanks. This has already been taken care of in the aws code branch. I removed it from this pull request.